content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "App Profile Cloning"
+metaTitle: "Palette Dev Engine App Profile Cloning"
+metaDescription: "Palette Dev Engine App Profile Cloning"
```

Comment:
The description is used by Google and affects our SEO score. Let's make sure we add more context and explain what this page is for.